draco/core/containers.py

```diff
@@ -1011,10 +1011,16 @@ def __init__(self, polarisation=True, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
 
+        self.attrs["nside"] = int(np.sqrt(len(self.index_map["pixel"]) / 12))
+
     @property
     def map(self):
         return self.datasets["map"]
 
+    @property
+    def nside(self):
+        return self.attrs["nside"]
+
```

Comment:
I don't see why this can't just be a dynamic property.
```suggestion
    @property
    def map(self):
        return self.datasets["map"]

    @property
    def nside(self):
        return int((len(self.index_map["pixel"]) //  12) ** 0.5)

```